components/ide/jetbrains/image/BUILD.yaml

```diff
@@ -51,7 +51,7 @@ packages:
         helm-component: workspace.desktopIdeImages.intellijLatest
       buildArgs:
         # "https://data.services.jetbrains.com/products?code=IIU&fields=distributions%2Clink%2Cname%2Creleases&_=$(date +%s)000"
-        JETBRAINS_BACKEND_URL: "https://download.jetbrains.com/idea/ideaIU-221.4994.44.tar.gz"
+        JETBRAINS_BACKEND_URL: "https://download.jetbrains.com/product?type=eap&distribution=linux&code=IIU"
```

Comment:
we actually need not eap type, but whatever is latest, i.e. it can be rc for instance

Reply:
I think you can drop `type=eap&` and it will do

Reply:
also remove comments above then

Reply:
Agreed :)

Let me update it.

Reply:
Actually dropping it did not work for GoLand, it pulls 2021.3 not EAP. We need to find a way to resolve really latest does not matter whether it is EAP, rc, released or something else

Reply:
We do have a way actually which we use in JB Nightly job: 
https://github.com/gitpod-io/gitpod/blob/d4bc3fdb523b6276ad38ec7305d61594de422210/.github/workflows/jetbrains-auto-update-template.yml#L49-L50

Reply:
It seems they only have 3 categories: `release`, `rc`, `eap`.

So we can use this link to download any latest, except `release`:
https://download.jetbrains.com/product?type=rc,eap&distribution=linux&code=GO

(ℹ️ The order of the `type` param doesn't affect the result. For example, `rc,eap` is the same as `eap,rc`)

What do you think?

Reply:
but release is latest for a month at least after rc

Reply:
Ah, listing all 3 will also work :D

https://download.jetbrains.com/product?type=release,rc,eap&distribution=linux&code=GO

Reply:
Could you update please? 🙏

Reply:
Updated 🚀